src/Core/NuGet/Core.NuGet-Beta/Package.nuspec

```diff
@@ -24,7 +24,7 @@
       </group>
       <group targetFramework="net46">
       </group>
-      <group targetFramework="dnxcore50">
+      <group targetFramework=".NETStandardApp,Version=v1.5">
```

Comment:
Why `netstandardapp`, not `netstandard`? It should work on any platform and cannot be run by itself.